lib/model/prefs.js

```diff
@@ -10,6 +10,7 @@ const eventForShowPref = (changePrefEvent) => {
     logger_dir: cliPrefs.logger.logDirectory,
     logger_location: cliPrefs.logger.logLocation,
     logger_blacklist: cliPrefs.logger.blacklist,
+    timeouts_session: cliPrefs.timeouts.session,
```

Comment:
We might need to decouple the model and output for `prefs`. What do you think, sir @prasadtalasila?
Well, it's not of concern with regard to this PR, can go in another one later.